lib/archivist/configuration.js

```diff
@@ -695,3 +695,20 @@ exports.setup = function (_logger, cb) {
 		return cb();
 	});
 };
+
+exports.connectVaults = function (_logger, cb) {
+	async.eachSeries(
+		Object.keys(persistentVaults),
+		function (vaultName, callback) {
+			const vault = persistentVaults[vaultName];
+
+			if (!vault.open) {
+				logger.error(`open method missing for vault ${vault.constructor.name}`);
+				return callback();
+			}
+
+			vault.open(callback);
```

Comment:
`connectVaults` and `open` ends up using two words for the same action; we should be able to connect/disconnect vaults one at a time too, so we should make sure the API naming convention is consistent

Reply:
Same as lib/archivist/index.js comment